### PR TITLE
Add project launch: We just released our rust teleconferencing system for free (free as in freedom) TWiR-504

### PR DESCRIPTION
**Pull request by darioalessandro**

### CODE REVIEW

File: draft/2023-07-19-this-week-in-rust.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ and just ask the editors to select the category.
 ### Newsletters
 
 ### Project/Tooling Updates
+* [We Just Released our Rust Teleconferencing System For Free (Free as in Freedom)](https://www.youtube.com/watch?v=KXfiWbCHlBw)
```
Comment (mariannegoldin) [inline draft/2023-07-19-this-week-in-rust.md:37]: Editing for length, adding link to the Rust codebase for readers' context/use
```suggestion
* [We Just Released our Rust Teleconferencing System for Free](https://www.youtube.com/watch?v=KXfiWbCHlBw) | [Repo](https://github.com/security-union/videocall-rs)
```

Comment (mariannegoldin) [inline draft/2023-07-19-this-week-in-rust.md:37]: Editing for length, adding link to the Rust codebase for readers' context/use
```suggestion
* [We Just Released our Rust Teleconferencing System for Free](https://www.youtube.com/watch?v=KXfiWbCHlBw) | [Repo](https://github.com/security-union/videocall-rs)
```

**Comment by mariannegoldin** [comment]
Having some issues updating this PR -- made a new PR #4561 with this update.

**Review by mariannegoldin — COMMENTED** [review]
Updated via #4561

File: draft/2023-07-19-this-week-in-rust.md (hunk 1 above)
Comment (darioalessandro) [reply draft/2023-07-19-this-week-in-rust.md:37]: Yes! this is perfect

**Pull request closed without merging**